src/js/view/controls/controlbar.js

```diff
@@ -246,6 +269,10 @@ export default class Controlbar {
         _model.change('cues', this.addCues, this);
         _model.change('altText', this.setAltText, this);
         _model.change('customButtons', this.updateButtons, this);
+        _model.change('state', () => {
+            // Check for change of position to counter race condition where state is updated before the current position
+            _model.once('change:position', this.checkDvrLiveEdge, this);
```

Comment:
@egreaves i used the once listener, not the on. We need to wait for the first change on position because state change occurs before position is updated therefore the check for isLive returns false. Is the comment i left in the code unclear ? If so i'll update it